src/core/InputSlider/index.tsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import { InputSliderExtraProps, StyledSlider } from "./style";
+
+/**
+ * @see https://v4.mui.com/components/slider/
+ */
+const InputSlider = (props: InputSliderExtraProps) => {
+  return <StyledSlider {...props} />;
+};
+
+export default InputSlider;
```

Comment:
Let's forwardRef 🎉 !

```suggestion
import React, { forwardRef } from "react";
import { InputSliderExtraProps, StyledSlider } from "./style";

/**
 * @see https://v4.mui.com/components/slider/
 */
const InputSlider = forwardRef<HTMLSpanElement, InputSliderExtraProps>(
  (props, ref) => {
    return <StyledSlider ref={ref} {...props} />;
  }
);

export default InputSlider;

```

Reply:
Ahh, sure thing. Thank you.